ibm/mas_devops/roles/suite_app_install/defaults/main.yml

```diff
@@ -19,3 +19,4 @@ mas_entitlement_key: "{{ lookup('env', 'MAS_ENTITLEMENT_KEY') }}"
 
 # MAS application configuration
 mas_app_id: "{{ lookup('env', 'MAS_APP_ID') }}"
+mas_app_plan: "{{ lookup('env', 'MAS_APP_PLAN') | default('full', true) }}"
```

Comment:
We shouldn't default this here ... there's no sensible default for all apps.

Reply:
Agree. Removed.